jobs/opi/templates/pre-start.erb

```diff
@@ -6,6 +6,8 @@ job_dir=/var/vcap/jobs/opi
 export KUBECONFIG=${job_dir}/config/kube.conf
 kubectl=/var/vcap/packages/kubectl/bin/kubectl
 
+$kubectl apply -f <($kubectl create namespace "<%= p('opi.kube_namespace') %>" --dry-run --save-config -o yaml)
```

Comment:
I'm trying to consume 0.0.16 on scf now and I noticed that kubectl package isn't included in the opi job. I will add it and test it now but I wonder how it works for you. We should better fix that too before we cut a new release. The rest of this PR is good.

Reply:
wait, we already create the namespace here: https://github.com/cloudfoundry-community/eirini-bosh-release/blob/master/jobs/configure-eirini-scf/templates/run.erb#L14

we check if it exists first but still, we should decide where we want this to run. Let me see how the testing of 0.0.16 goes and I'll post my findings here. Maybe we need the same fix anyway for scf (create the namespace before the job runs).

Reply:
We let helm create the namespace: https://github.com/SUSE/scf/blob/develop/chart-parts/eirini-namespace.yaml